astro-docs/src/content/docs/concepts/common-tasks.mdoc

```diff
@@ -1,9 +1,12 @@
 ---
-title: Common Tasks
+title: Common Task Configurations
 description: Learn about standard task naming conventions in Nx projects, including build, serve, test, and lint tasks, for consistent project configuration.
 keywords: [build, serve, test, lint]
 sidebar:
   order: 1
+  label: Common Tasks
+filter: "type:Concepts"
+weight: 5.0
 ---
```

Comment:
updated title here to try to add some more weight in relevancy but any more weight and it quickly overpowers the search results for unrelated things this one we'll need to play with more I think in increasing the content to help saturate